lua/FTerm/terminal.lua

```diff
@@ -196,4 +199,10 @@ function Terminal:toggle()
     end
 end
 
+-- Terminal:run is used to (open and) run commands to terminal window
+function Terminal:run(command)
+    self:open()
+    api.nvim_chan_send(self.jobid, command)
+end
```

Comment:
I am concerned with calling `self:open()` call here, what will happen if the window is still open. Have you considered multiple `t:run()` functions like the following, where run can be called via some keybindings.

```lua
local t = require('FTerm.terminal'):new():setup()

t:run("echo 123")

t:run("cat foo.txt")

t:run("man find")
```

Reply:
Except for the missing `\n` at the end of each statement, it works just fine on my system. Doesn't seem to be an issue on my end.

But putting up an guard to check against "double" opening of windows is perhaps a good idea, but perhaps place it somewhere in `open`? Like running `t:open(); t:open(); t:open()` should be equally unproblematic as running `t.run(...)` three times.

I'll try to add that.

Reply:
@numToStr, guard is now in place.